native/src/main/java/io/ballerina/stdlib/sql/parameterprocessor/AbstractResultParameterProcessor.java

```diff
@@ -215,6 +215,9 @@ public abstract Object processCustomOutParameters(
     public abstract Object processCustomTypeFromResultSet(ResultSet resultSet, int columnIndex,
                                                            ColumnDefinition columnDefinition) throws ApplicationError;
 
+    public abstract Object getAndConvertXml(ResultSet resultSet, int columnIndex, int sqlType, Type balType)
```

Comment:
```suggestion
    public Object processXmlResult(ResultSet resultSet, int columnIndex, int sqlType, Type balType) throws ApplicationError, SQLException {
        SQLXML sqlxml = resultSet.getSQLXML(columnIndex);
         return convertXml(sqlxml, sqlType, balType);
     }
     public abstract Object convertXml(sqlxml, sqlType, ballerinaType);
```

Reply:
updated in https://github.com/ballerina-platform/module-ballerina-sql/pull/329/commits/ec7082e053a0528f35ea6b01e858df0b0ba0fb6f